test/python/asset/main.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
+import os
+from constructs import Construct
+from cdktf import App, TerraformStack, Testing, TerraformAsset
+from imports.null import NullProvider, Resource
+
+class MyStack(TerraformStack):
+    def __init__(self, scope: Construct, ns: str):
+        super().__init__(scope, ns)
+
+        NullProvider(self, "null")
+        Resource(self, "null-resource")
+        self.add_override('terraform.backend', {
```

Comment:
If specifying a remote backend is useful here, should probably use the code defined version rather than the override. If isn't really part of the test, might be best to just remove.

Reply:
will remove it 👍